pages/docs/concepts/server.md

```diff
@@ -0,0 +1,6 @@
+
+## What is a Server?
+APIs use servers to receive requests from a user and responds back with the required information. In a nutshell, a server is a middle man between a user and the stored information inside a database. The communication between a user and the database happens in the form of HTTP messages. One thing to note, servers cannot make any requests, and the user can’t respond as all interactions are instigated by the user.
+
+## Purpose of Servers in Event-Driven Architectures
+Servers play an important role in event-driven architecture, as they create a relationship between an application and clients or/consumers. When designing and setting up an event-driven application, servers are in charge of delivering asynchronous messages to the consumer. With the integration of Websocket APIs, servers can create and maintain the server-client connection or server to server connections.
```

Comment:
> With the integration of Websocket APIs

Brokers accept connections using different protocols. It can be Websocket but any other as well (Plain TCP). In the line of  what @fmvilas mentioned earlier:
> Partially true as well. In the case of AsyncAPI, in most cases, communication happens using a messaging protocol like MQTT, AMQP, Kafka, WebSocket, etc.